### PR TITLE
Added a generate load script for real environment testing.

### PR DESCRIPTION
**Pull request by VitaliiYerys**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

File: devlab/tests/generate_load_real_env.py [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+
+
+class Prerequisites():
+    def __init__(self, cloud_prefix='SRC'):
+        self.username = os.environ['%s_OS_USERNAME' % cloud_prefix]
+        self.password = os.environ['%s_OS_PASSWORD' % cloud_prefix]
+        self.tenant = os.environ['%s_OS_TENANT_NAME' % cloud_prefix]
+        self.auth_url = os.environ['%s_OS_AUTH_URL' % cloud_prefix]
+        self.image_endpoint = os.environ['%s_OS_IMAGE_ENDPOINT' % cloud_prefix]
+        self.neutron_endpoint = os.environ['%s_OS_NEUTRON_ENDPOINT'
```
Comment (roman-verchikov) [inline devlab/tests/generate_load_real_env.py]: glance and neutron endpoints can be retrieved from keystone (unless you want to use specific endpoints)

File: devlab/tests/generate_load_real_env.py [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+        return [x for x in volumes if x.display_name == volume_name][0].id
+
+    def get_volume_snapshot_id(self, snapshot_name):
+        snapshots = self.cinderclient.volume_snapshots.list(
+            search_opts={'all_tenants': 1})
+        return [x for x in snapshots if x.display_name == snapshot_name][0].id
+
+    def check_vm_state(self, srv):
+        while srv.status != 'ACTIVE':
+                time.sleep(2)
```
Comment (roman-verchikov) [inline devlab/tests/generate_load_real_env.py]: overindented, use 4 spaces please

File: devlab/tests/real_env_conf.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+cinder_volumes = [
+    {'name': 'cinder_volume1', 'size': 1},
+    {'name': 'cinder_volume2', 'size': 1,
+     'server_to_attach': 'server2', 'device': '/dev/vdb'}
+]
+
+# Cinder snapshots to create/delete
+cinder_snapshots = [
+    # Commented because of unimplemented error in nfs driver for grizzly.
+    # {'display_name': 'snapsh1', 'volume_id': 'cinder_volume1'}
```
Comment (roman-verchikov) [inline devlab/tests/real_env_conf.py]: We should generate cinder snapshots in prod, this was commented only for devenv where NFS driver does not support snapshots.

**Comment by roman-verchikov** [comment]
Please address inlines

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

File: devlab/tests/real_env_conf.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+# Cinder images to create/delete
+cinder_volumes = [
+    {'name': 'cinder_volume1', 'size': 1},
+    {'name': 'cinder_volume2', 'size': 1,
+     'server_to_attach': 'server2', 'device': '/dev/vdb'}
+]
+
+# Cinder snapshots to create/delete
+cinder_snapshots = [
+    # Commented because of unimplemented error in nfs driver for grizzly.
```
Comment (roman-verchikov) [inline devlab/tests/real_env_conf.py]: I guess you can remove this comment, it's not valid for this particular script

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

**Comment by roman-verchikov** [comment]
Looks ok, +1

**Comment by roman-verchikov** [comment]
Need one more reviewer to merge

**Comment by ntymtsiv** [comment]
Please do not merge for now

**Comment by roman-verchikov** [comment]
Looks ok to me. @ntymtsiv do you have any comments?

**Comment by ntymtsiv** [comment]
lgfm, +1

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**